packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -21,6 +21,10 @@
   "download_app_right_pane_paradym_qrcode_text": "Paradym Wallet",
   "download_app_right_pane_sphereon_qrcode_text": "Sphereon Wallet",
   "download_app_right_pane_title": "Compliant SSI Wallets",
+  "sphereon_wallet_right_pane_title": "Download the Wallet",
+  "sphereon_wallet_right_pane_paragraph": "",
```

Comment:
we need text here or we need to remove the localization item

Reply:
The empty ones are removed, the page code is now using
```
        paragraphResourceId? : string
        qrTextResourceId? : string
```
(when populated)